Editor/Window/AwsUserProfilesPage.cs

```diff
@@ -0,0 +1,269 @@
+            l.SetElementText("UserProfilePageBootstrapBucketText", Strings.UserProfilePageBootstrapBucketText);
+            l.SetElementText("UserProfilePageBootstrapBucketUnsetText", Strings.UserProfilePageBootstrapBucketUnsetText);
+            l.SetElementText("LabelBootstrapRegion", Strings.LabelBootstrapRegion);
+            l.SetElementText("UserProfilePageBootstrapStatusText", Strings.UserProfilePageBootstrapStatusText);
+            l.SetElementText("UserProfilePageBootstrapWarningText", Strings.UserProfilePageBootstrapWarningText);
+            l.SetElementText("UserProfilePageBootstrapProfilePlaceholderText", Strings.UserProfilePageBootstrapProfilePlaceholderText);
+            l.SetElementText("UserProfilePageBootstrapPricingInfoText", Strings.UserProfilePageBootstrapPricingInfoText);
+            l.SetElementText("UserProfilePageBootstrapPricingFreeTierText", Strings.UserProfilePageBootstrapPricingFreeTierText);
+            l.SetElementText("UserProfilePageBootstrapHelpLink", Strings.UserProfilePageBootstrapHelpLink);
+            l.SetElementText("UserProfilePageBootstrapStartButton", Strings.UserProfilePageBootstrapStartButton);
+            l.SetElementText("UserProfilePageBootstrapAnotherProfileButton", Strings.UserProfilePageBootstrapAnotherProfileButton);
+            l.SetElementText("UserProfilePageBootstrapAnotherBucketButton", Strings.UserProfilePageBootstrapAnotherBucketButton);
+        }
+
+        private void SetupButtonCallbacks()
+        {
+            _container.Q<Button>("UserProfilePageAccountCardNoAccountButton").RegisterCallback<ClickEvent>(_ => OpenLink(""));
+            _container.Q<Button>("UserProfilePageBootstrapAnotherProfileButton").RegisterCallback<ClickEvent>(_ =>
+            {
+                ChangeWizard(_container.Q<VisualElement>("AddNewProfile"));
+            });
+            _container.Q<Button>("UserProfilePageAccountNewProfileCreateButton").RegisterCallback<ClickEvent>(_ =>
+            {
+                if (SaveProfile())
+                {
+                    var targetWizard = _container.Q<VisualElement>("BootstrapMenu");
+                    ChangeWizard(targetWizard);
+                }
+                else
+                {
+                    Debug.Log("Error");
+                }
+            });
+            _container.Q<Button>("UserProfilePageAccountNewProfileCancelButton").RegisterCallback<ClickEvent>(_ =>
+            {
+                ClearCredentials();
+                SetupBootMenu();
+            });
+            _container.Q<Button>("UserProfilePageBootstrapStartButton").RegisterCallback<ClickEvent>(_ =>
+            {
+                BootstrapSettings.RefreshBucketName();
+                OpenS3Popup(BootstrapSettings.BucketName);
+            });
+            _container.Q<Button>("UserProfilePageBootstrapAnotherBucketButton").RegisterCallback<ClickEvent>(_ =>
+            {
+                // TODO: Add or find functionality for this
```

Comment:
Please address this TODO as it seems relevant to this PR

Reply:
Added